src/plugins/intel_cpu/src/node.cpp

```diff
@@ -1352,7 +1352,6 @@ Node* Node::NodesFactory::create(const std::shared_ptr<ov::Node>& op, const Grap
 }
 
 bool Node::canBePerformedAsScaleShift(const Node *parentNode) const {
-#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
Could you please elaborate why we need to have these changes in this PR?

Reply:
Thanks Alexandra for the comment! The behavor of canBePerformedAsScaleShift in node needs to align with its counterpart in snippets_mark_skipped. Before revision, Node::canBePerformedAsScaleShift always return false on arm. So there will be cases, where a candidate node is neither be tokenzied for Snippets, nor be fused as post ops.

Reply:
I'd say that `SnippetsMarkSkipped` is based on `Node` supports and impls. We should update `SnippetsMarkSkipped` based on operation supports, optimization transformation in `GraphOptimizer`. So you don't need to touch the method `Node::canBePerformedAsScaleShift`. Moreover, this method is used in [other](https://github.com/xuchen-intel/openvino/blob/28d718426780007c8bd5c80b2d6f3506cb33a87d/src/plugins/intel_cpu/src/nodes/deconv.cpp#L331) nodes. If you update `canBePerformedAsScaleShift`, it can affect other nodes.
So if `ARM` doesn't support `canBePerformedAsScaleShift`, `SnippetsMarkSkipped` shouldn't verify these usecase

Reply:
Applied. Thanks Alexandra!